changelog/deprecateNonPropertyAssignment.dd

```diff
@@ -0,0 +1,16 @@
+the property assignment to non-@property functions has been deprecated.
```

Comment:
I think we should do this: `Properties with exactly one defaulted value cannot be assigned to.` Is that correct?